quilt/run.go

```diff
@@ -0,0 +1,155 @@
+package main
```

Comment:
You're going to hate this comment....

Since the Ray project is so python focused, I wonder if it makes sense for their run script to be written in python instead of go?  I defer to @robertnishihara @massie on this.  I love go, but it seems a bit out of place in this repo.